app/User.php

```diff
@@ -35,4 +35,8 @@ public function hasRole($role) {
         }
         return false;
     }
+
+    public function getModel(){
+        return Member::where('crsid',$this->crsid)->first();
```

Comment:
You might want a space here! `('crsid', $this->crsid)`

Also, the idea here is correct! Laravel/Eloquent has something for this though, check out https://laravel.com/docs/5.3/eloquent-relationships#one-to-one

```
class User extends ...
{
    ....
    public function member()
    {
        return $this->hasOne('App\Member', 'crsid', 'crsid');
    }
}

```